jaxrs-clients/src/main/java/com/palantir/remoting1/jaxrs/FeignJaxRsClientBuilder.java

```diff
@@ -96,7 +96,7 @@
                     .build(),
                     ConnectionSpec.CLEARTEXT);
 
-    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.guavaJdk7();
+    private static final ObjectMapper OBJECT_MAPPER = ObjectMappers.guavaJdk8();
```

Comment:
one thought i had i that it might be useful/more future proof to add a "overrideObjectMapper" method to the builders, any thoughts?